master/internal/experiment.go

```diff
@@ -306,6 +306,12 @@ func (e *experiment) Receive(ctx *actor.Context) error {
 	case trialExitedEarly:
 		ops, err := e.searcher.TrialExitedEarly(msg.trialID)
 		e.processOperations(ctx, ops, err)
+	case sendNextWorkload:
```

Comment:
Maybe a comment here, since `sendNextWorkload` doesn't have much meaning in the context of the experiment actor.

Reply:
done